llvm/docs/conf.py

```diff
@@ -222,6 +222,7 @@
 # If false, no module index is generated.
 # latex_domain_indices = True
 
+numfig = True
```

Comment:
nit: can you add a comment describing the purpose of enabling this?

Reply:
Missed originally, should be addressed in 309a881dccb82bf1f101cf138bee3e7400968ed8